rpc/lib/doc.go

```diff
@@ -97,7 +97,7 @@ Each route is available as a GET request, as a JSONRPCv2 POST request, and via J
 
 # Examples
 
-* [Tendermint](https://github.com/tendermint/tendermint/blob/master/rpc/core/routes.go)
-* [tm-monitor](https://github.com/tendermint/tools/blob/master/tm-monitor/rpc.go)
+* [Tendermint](https://github.com/tendermint/tendermint/blob/develop/rpc/core/routes.go)
```

Comment:
why change to `develop`?

Reply:
for consistency

Reply:
doc should point to the latest stable release - master

Reply:
fixed